src/components/AppShell.tsx

```diff
@@ -0,0 +1,26 @@
+import { AppShell, Navbar, Header } from '@mantine/core';
+import NewsComponent from './NewsComponent';
+import TodoList from './TodoList';
+import WeatherComponent from './WeatherComponent';
+
+function Shell() {
+  return (
+    <AppShell
+      padding="md"
+      navbar={
+        <Navbar width={{ base: 300 }} height={800} p="xs">
+          <TodoList />
+        </Navbar>
```

Comment:
I would move the ToDo out of the nav bar, although it is placed correctly in the UI it is no the proper use of this property in AppShell